frontend/test/lit/test_gradient.py

```diff
@@ -31,7 +31,7 @@ def f(x: float):
         qml.RX(x, wires=0)
         return qml.expval(qml.PauliY(0))
 
-    # CHECK: gradient.grad "fd" @f({{%[0-9]+}}) {diffArgIndices = dense<0> : tensor<1xi64>, finiteDiffParam = 9.9999999999999995E-8 : f64} : (tensor<f64>) -> tensor<f64>
+    # CHECK: gradient.grad "fd" @f(%cst) {diffArgIndices = dense<0> : tensor<1xi64>, finiteDiffParam = 9.9999999999999995E-8 : f64} : (tensor<f64>) -> tensor<f64>
```

Comment:
The problem here is if the printing changes again we will have to update these tests as well. A longer-lived fix would be to make the SSA regex more permissive to also allow letters.
It's a shame FileCheck doesn't support regex shortcuts like `\d` or `\w` :/